FastBank.Services/MessageService/MessageService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using FastBank.Domain;
+using FastBank.Domain.RepositoryInterfaces;
+using FastBank.Infrastructure.DTOs;
+using FastBank.Infrastructure.Repository;
+
+namespace FastBank.Services.MessageService
+{
+    public class MessageService : IMessageService
+    {
+        private readonly IMessageRepository _messageRepo;
+
+        public MessageService()
+        {
+            _messageRepo = new MessageRepository();
+        }
+
+        public void AddMessage(Message message)
+        {
+            _messageRepo.Add(message);
+        }
+
+        public void GetMessages(Customer customer)
+        {
+            var messages = _messageRepo.GetCustomerMessages(customer);
+
+            foreach (var message in messages)
+            {
+                Console.WriteLine($"Subject: {message.Subject}");
+                Console.WriteLine($"Text: {message.Text}");
+                Console.WriteLine($"Status: {message.Status}");
+                Console.WriteLine(new string('*', Console.WindowWidth));
+                Console.WriteLine(new string(' ', Console.WindowWidth));
+            }
+            Console.ReadKey();
+        }
+
+        public void AddMessage(string subject,
+                               string text,
+                               MessageStatuses status,
+                               MessageType type,
+                               Customer sender,
+                               Customer? receiver,
+                               Roles receiverRole,
+                               Message basedOnMessage)
+        {
+            Message message = new Message(Guid.NewGuid(), sender, receiver, receiverRole, text, subject, null, status, type);
+            _messageRepo.Add(message);
+        }
+
+        public Message InputMessage(Customer customer)
+        {
+            Console.WriteLine("Please input message subject:");
+            var subject = Console.ReadLine();
+
+            Console.WriteLine("Please text message");
+            var text = Console.ReadLine();
+
+            var message = new Message(Guid.NewGuid(), customer, null, Roles.CustomerService,
+                                      text, subject, null, MessageStatuses.Sent, MessageType.Inquery);
+            
+            AddMessage(message);
+            return message;
+        }
```

Comment:
Please refactor. Use only one method for creating the message. In InputMessage you take the input from the user and then you create the message. Afterwards, you are going to AddMessage and again you create a new message. Why? Get the input, build the message and save it to the repo. That's it. Please, keep your code and follow the following priciple - KISS. Reference: https://dev.to/kwereutosu/the-k-i-s-s-principle-in-programming-1jfg